tests/Feature/WebServiceSoapServiceCallerTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use Illuminate\Foundation\Testing\WithFaker;
+use Mockery;
+use ProcessMaker\Contracts\SoapClientInterface;
+use ProcessMaker\Managers\WebServiceSoapServiceCaller;
+use Tests\TestCase;
+
+class WebServiceSoapServiceCallerTest extends TestCase
+{
+    use WithFaker;
+
+    /**
+     *
+     * @var WebServiceSoapServiceCaller
+     */
+    private $manager;
+
+    protected function setUp(): void
+    {
+        parent::__construct();
+        $this->manager = new WebServiceSoapServiceCaller;
+        $this->app = app();
```

Comment:
@nolanpro I have to add this line to have access to the `app` object. Which is required to use helpers like: `$this->instance` and `$this->mock`